### ISSUE TITLE
[Feature request] Now for something crazy: Android TV

### ISSUE THREAD

**Issue by frauhottelmann** [anthonycr/Lightning-Browser#380]
I just got a Nvidia Shield TV running Android TV. Android TV doesn't have a build in browser (strangely enough), but it of course has webview. I've tried a few so far and none are really good. Either because you can't really navigate their UIs and or the webpages (Chrome, Firefox, Opera or Lightning) or they can't download stuff (the one available Android TV browser or Opera's TV browser).

So my feature request would be a UI that is navigable with directional keys only. In tablet ui mode I can easily get to the back and forward keys and the menu button, but I can't select the address bar. Downloading works, but since there is no notification tray of any kind you just have to wait and guess when it's finished (that's the reason I am using ES File Explorer's download mode, which displays a pop up). I am only hoping for directional button support. It works fine with a mouse btw.

**Comment by jeffkrebs** [comment]
I'm reading online that people are plugging a wired USB mouse into the Shield and controlling the browser that way:

https://www.youtube.com/watch?v=ujLKtg5mdLI

A corded, optical micro-mouse would certainly provide more options and flexibility than directional buttons:
http://www.amazon.com/dp/B00Y20UI1K?psc=1
http://www.amazon.com/Microsoft-Compact-Optical-Mouse-Business/dp/B005058AHU/ref=sr_1_1?s=pc&ie=UTF8&qid=1455591923&sr=1-1&keywords=compact+corded+mouse

**Comment by frauhottelmann** [comment]
Yes, it does work with a mouse, but that's not always an option, espacially when I am sitting on the couch with the controller only. I have a keyboard mousepad combo now (from Rapoo) abd it works great.
